src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -506,14 +510,21 @@ public ListBoxModel doFillRepositoryItems(@AncestorInPath SCMSourceOwner context
                     for (GHRepository repo : user.listRepositories()) {
                         result.add(repo.getName());
                     }
-                    return result;
+                    return model(result);
                 }
             } catch (SSLHandshakeException he) {
                 LOGGER.log(Level.SEVERE, he.getMessage());
             } catch (IOException e) {
                 LOGGER.log(Level.SEVERE, e.getMessage());
             }
-            return result;
+            return model(result);
+        }
+        private static ListBoxModel model(Collection<String> items) {
+            ListBoxModel model = new ListBoxModel();
+            for (String item : items) {
+                model.add(item);
```

Comment:
There is a `ListBoxModel.<init>(Collection)` but it takes `Option`s, not `String`s, and those are not `Comparable`.